### PR TITLE
[zh]sync changes from english.

### PR DESCRIPTION
**Pull request by yuandongx**
Sycn chages.
+ content/zh/docs/reference/kubectl/overview.md

Related Issues kubernetes/website#25247

### CODE REVIEW

File: content/zh/docs/reference/kubectl/overview.md [hunk 1]
```diff
@@ -25,14 +25,27 @@ card:
...
+The kubectl command line tool lets you control Kubernetes clusters.
+For configuration, `kubectl` looks for a file named `config` in the `$HOME/.kube` directory.
+You can specify other [kubeconfig](/docs/concepts/configuration/organize-cluster-access-kubeconfig/)
+files by setting the KUBECONFIG environment variable or by setting the
+[`--kubeconfig`](/docs/concepts/configuration/organize-cluster-access-kubeconfig/) flag.
 -->
-Kubectl 是一个命令行接口，用于对 Kubernetes 集群运行命令。`kubectl` 在 $HOME/.kube 目录中寻找一个名为 config 的文件。您可以通过设置环境变量 KUBECONFIG 或设置 [`--kubeconfig`](/zh/docs/concepts/configuration/organize-cluster-access-kubeconfig/) 参数指定其它 [kubeconfig](/zh/docs/concepts/configuration/organize-cluster-access-kubeconfig/) 文件。
+Kubectl 命令行工具使你能够管理 Kubernetes 集群。
+`kubectl` 在 $HOME/.kube 目录中查找一个名为 `config` 的文件。
```
Comment (helight) [inline content/zh/docs/reference/kubectl/overview.md:35]:
```suggestion
`kubectl` 在 `$HOME/.kube` 目录中查找一个名为 `config` 的配置文件。
```

File: content/zh/docs/reference/kubectl/overview.md [hunk 2]
```diff
@@ -25,14 +25,27 @@ card:
...
+The kubectl command line tool lets you control Kubernetes clusters.
+For configuration, `kubectl` looks for a file named `config` in the `$HOME/.kube` directory.
+You can specify other [kubeconfig](/docs/concepts/configuration/organize-cluster-access-kubeconfig/)
+files by setting the KUBECONFIG environment variable or by setting the
+[`--kubeconfig`](/docs/concepts/configuration/organize-cluster-access-kubeconfig/) flag.
 -->
-Kubectl 是一个命令行接口，用于对 Kubernetes 集群运行命令。`kubectl` 在 $HOME/.kube 目录中寻找一个名为 config 的文件。您可以通过设置环境变量 KUBECONFIG 或设置 [`--kubeconfig`](/zh/docs/concepts/configuration/organize-cluster-access-kubeconfig/) 参数指定其它 [kubeconfig](/zh/docs/concepts/configuration/organize-cluster-access-kubeconfig/) 文件。
+Kubectl 命令行工具使你能够管理 Kubernetes 集群。
```
Comment (helight) [inline content/zh/docs/reference/kubectl/overview.md:34]:
```suggestion
可以使用 Kubectl 命令行工具来管理 Kubernetes 集群。
```

File: content/zh/docs/reference/kubectl/overview.md [hunk 3]
```diff
@@ -25,14 +25,27 @@ card:
...
-This overview covers `kubectl` syntax, describes the command operations, and provides common examples. For details about each command, including all the supported flags and subcommands, see the [kubectl](/docs/reference/generated/kubectl/kubectl-commands/) reference documentation. For installation instructions see [installing kubectl](/docs/tasks/kubectl/install/).
+This overview covers `kubectl` syntax, describes the command operations, and provides common examples.
+For details about each command, including all the supported flags and subcommands, see the
+[kubectl](/docs/reference/generated/kubectl/kubectl-commands/) reference documentation.
+For installation instructions see [installing kubectl](/docs/tasks/tools/install-kubectl/).
 -->
-本文概述了 `kubectl` 语法和命令操作描述，并提供了常见的示例。有关每个命令的详细信息，包括所有受支持的参数和子命令，请参阅 [kubectl](/docs/reference/generated/kubectl/kubectl-commands/) 参考文档。有关安装说明，请参见 [安装 kubectl](/zh/docs/tasks/tools/install-kubectl/) 。
+本文概述了 `kubectl` 语法和命令操作描述，并提供了常见的示例。
+有关每个命令的详细信息，包括所有受支持的参数和子命令，
+请参阅 [kubectl](/docs/reference/generated/kubectl/kubectl-commands/) 参考文档。
```
Comment (helight) [inline content/zh/docs/reference/kubectl/overview.md:47]:
```suggestion
请参阅 [kubectl](/zh/docs/reference/generated/kubectl/kubectl-commands/) 参考文档。
```

File: content/zh/docs/reference/kubectl/overview.md [hunk 4]
```diff
@@ -467,9 +513,9 @@ metadata.name metadata.resourceVersion
 ```
 
 <!--
-The result of running either command is:
+The result of running either command is similar to:
 -->
-运行任何一个命令的结果是:
+运行任何一个命令的结果类似如下:
```
Comment (helight) [inline content/zh/docs/reference/kubectl/overview.md:518]:
```suggestion
运行任何一个命令的结果类似于:
```

File: content/zh/docs/reference/kubectl/overview.md [hunk 5]
```diff
@@ -511,13 +557,13 @@ kubectl get pods <pod-name> --server-print=false
 ```
 
 <!--
-Output looks like this:
+The output is similar to:
 -->
-输出如下：
+输出类似如下：
```
Comment (helight) [inline content/zh/docs/reference/kubectl/overview.md:562]:
```suggestion
输出类似于：
```

File: content/zh/docs/reference/kubectl/overview.md [hunk 6]
```diff
@@ -782,38 +851,41 @@ cat ./kubectl-hello
...
-sudo chmod +x ./kubectl-hello
+sudo chmod a+x ./kubectl-hello
 
 # 并将其移动到路径中的某个位置
 sudo mv ./kubectl-hello /usr/local/bin
+sudo chown root:root /usr/local/bin
 
-# 我们现在已经创建并"安装"了一个 kubectl 插件。
-# 我们可以开始使用我们的插件，从 kubectl 调用它，就像它是一个常规命令一样
+# 你现在已经创建并"安装"了一个 kubectl 插件。
```
Comment (helight) [inline content/zh/docs/reference/kubectl/overview.md:860]:
```suggestion
# 你现在已经创建并"安装了"一个 kubectl 插件。
```

File: content/zh/docs/reference/kubectl/overview.md [hunk 7]
```diff
@@ -782,38 +851,41 @@ cat ./kubectl-hello
...
+# 你可以开始使用我们的插件，从 kubectl 调用它，就像它是一个常规命令一样
```
Comment (helight) [inline content/zh/docs/reference/kubectl/overview.md:861]:
```suggestion
# 你可以开始使用这个插件，从 kubectl 调用它，就像它是一个常规命令一样
```

File: content/zh/docs/reference/kubectl/overview.md [hunk 8]
```diff
@@ -935,6 +1018,10 @@ To find out more about plugins, take a look at the [example cli plugin](https://
 
 
 <!--
-Start using the [kubectl](/docs/reference/generated/kubectl/kubectl-commands/) commands.
+* Start using the [kubectl](/docs/reference/generated/kubectl/kubectl-commands/) commands.
+
+* To find out more about plugins, take a look at the [example cli plugin](https://github.com/kubernetes/sample-cli-plugin).
 -->
-开始使用 [kubectl](/docs/reference/generated/kubectl/kubectl-commands/) 命令。
+* 开始使用 [kubectl](/docs/reference/generated/kubectl/kubectl-commands/) 命令。
```
Comment (helight) [inline content/zh/docs/reference/kubectl/overview.md:1025]:
```suggestion
* 开始使用 [kubectl](/zh/docs/reference/generated/kubectl/kubectl-commands/) 命令。
```

File: content/zh/docs/reference/kubectl/overview.md (hunk 1 above)
Comment (yuandongx) [reply content/zh/docs/reference/kubectl/overview.md:35]: Thanks,done

File: content/zh/docs/reference/kubectl/overview.md (hunk 2 above)
Comment (yuandongx) [reply content/zh/docs/reference/kubectl/overview.md:34]: Thanks, done

File: content/zh/docs/reference/kubectl/overview.md (hunk 6 above)
Comment (yuandongx) [reply content/zh/docs/reference/kubectl/overview.md:860]: Thanks, done.

File: content/zh/docs/reference/kubectl/overview.md (hunk 7 above)
Comment (yuandongx) [reply content/zh/docs/reference/kubectl/overview.md:861]: Thanks,done.

File: content/zh/docs/reference/kubectl/overview.md (hunk 8 above)
Comment (yuandongx) [reply content/zh/docs/reference/kubectl/overview.md:1025]: Thanks, done.

File: content/zh/docs/reference/kubectl/overview.md [hunk 9]
```diff
@@ -143,70 +156,102 @@ The following table includes short descriptions and the general syntax for all o
...
 `proxy`        | `kubectl proxy [--port=PORT] [--www=static-dir] [--www-prefix=prefix] [--api-prefix=prefix] [flags]` | 运行 Kubernetes API 服务器的代理。
 `replace`        | `kubectl replace -f FILENAME` | 从文件或标准输入中替换资源。
-`rolling-update`    | <code>kubectl rolling-update OLD_CONTROLLER_NAME ([NEW_CONTROLLER_NAME] --image=NEW_CONTAINER_IMAGE &#124; -f NEW_CONTROLLER_SPEC) [flags]</code> | 通过逐步替换指定的副本控制器及其 pod 来执行滚动更新。
+`rollout`    | `kubectl rollout SUBCOMMAND [options]` | 管理资源的部署。有效的资源类型包括：部署，守护程序集和状态集。
 `run`        | <code>kubectl run NAME --image=image [--env="key=value"] [--port=port] [--dry-run=server &#124; client &#124; none] [--overrides=inline-json] [flags]</code> | 在集群上运行指定的镜像。
 `scale`        | <code>kubectl scale (-f FILENAME &#124; TYPE NAME &#124; TYPE/NAME) --replicas=COUNT [--resource-version=version] [--current-replicas=count] [flags]</code> | 更新指定副本控制器的大小。
-`stop`        | `kubectl stop` | 不推荐：相反，请参阅 kubectl delete。
+`set`    | `kubectl set SUBCOMMAND [options]` | 配置应用程序资源。
+`taint`    | `kubectl taint NODE NAME KEY_1=VAL_1:TAINT_EFFECT_1 ... KEY_N=VAL_N:TAINT_EFFECT_N [options]` | 更新一个或多个节点上的污点。
+`top`    | `kubectl top [flags] [options]` | 显示资源（CPU /内存/存储）的使用情况。
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/overview.md]:
```suggestion
`top`    | `kubectl top [flags] [options]` | 显示资源（CPU/内存/存储）的使用情况。
```

File: content/zh/docs/reference/kubectl/overview.md [hunk 10]
```diff
@@ -632,11 +678,17 @@ kubectl get pods --field-selector=spec.nodeName=server01
...
+
 # Display the details of the pod with name <pod-name>.
+kubectl describe pods/<pod-name>
 
 # Display the details of all the pods that are managed by the replication controller named <rc-name>.
 # Remember: Any pods that are created by the replication controller get prefixed with the name of the replication controller.
+kubectl describe pods <rc-name>
+
 # Describe all pods, not including uninitialized ones
+kubectl describe pods
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/overview.md]: 705-707 行的例子不存在了，要删除。

File: content/zh/docs/reference/kubectl/overview.md [hunk 11]
```diff
@@ -680,10 +732,17 @@ the pods running on it, the events generated for the node etc.
...
-// Delete all pods, including uninitialized ones.
+kubectl delete -f pod.yaml
+
+# Delete all the pods and services that have the label '<label-key>=<label-value>'.
+kubectl delete pods,services -l <label-key>=<label-value>
+
+# Delete all pods, including uninitialized ones.
+kubectl delete pods --all
+```
+
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/overview.md]: 752-753 的例子要删掉。
755-756 的例子要更新。

File: content/zh/docs/reference/kubectl/overview.md [hunk 12]
```diff
@@ -707,8 +766,13 @@ kubectl delete pods --all
 
 <!--
 # Get output from running 'date' from pod <pod-name>. By default, output is from the first container.
+kubectl exec <pod-name> -- date
+
 # Get output from running 'date' in container <container-name> of pod <pod-name>.
+kubectl exec <pod-name> -c <container-name> -- date
+
 # Get an interactive TTY and run /bin/bash from pod <pod-name>. By default, output is from the first container.
+kubectl exec -ti <pod-name> -- /bin/bash
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/overview.md]: 783 行的命令不对。
786 行的命令不对。

File: content/zh/docs/reference/kubectl/overview.md [hunk 13]
```diff
@@ -781,39 +850,42 @@ cat ./kubectl-hello
 # 这个插件打印单词 "hello world"
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/overview.md]: 848 行没有更新

File: content/zh/docs/reference/kubectl/overview.md [hunk 14]
```diff
@@ -781,39 +850,42 @@ cat ./kubectl-hello
...
 <!--
-```
-# we can "uninstall" a plugin, by simply removing it from our PATH
+```shell
+# You can "uninstall" a plugin, by removing it from the folder in your
+# $PATH where you placed it
 sudo rm /usr/local/bin/kubectl-hello
 ```
 -->
 ```
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/overview.md]: 874 漏了 'shell'

File: content/zh/docs/reference/kubectl/overview.md [hunk 15]
```diff
@@ -830,19 +903,18 @@ The following kubectl-compatible plugins are available:
...
-# plugins, for example
-sudo chmod -x /usr/local/bin/kubectl-foo
+`kubectl plugin list` also warns you about plugins that are not
+executable, or that are shadowed by other plugins; for example:
+```shell
+sudo chmod -x /usr/local/bin/kubectl-foo # remove execute permission
 kubectl plugin list
 ```
 -->
 ```
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/overview.md]: 913 shell

**Comment by tengqm** [comment]
Why is this closed? I have spent about one hour reviewing it ...

File: content/zh/docs/reference/kubectl/overview.md [hunk 16]
```diff
@@ -25,14 +25,27 @@ card:
...
+The kubectl command line tool lets you control Kubernetes clusters.
+For configuration, `kubectl` looks for a file named `config` in the `$HOME/.kube` directory.
+You can specify other [kubeconfig](/docs/concepts/configuration/organize-cluster-access-kubeconfig/)
+files by setting the KUBECONFIG environment variable or by setting the
+[`--kubeconfig`](/docs/concepts/configuration/organize-cluster-access-kubeconfig/) flag.
 -->
-Kubectl 是一个命令行接口，用于对 Kubernetes 集群运行命令。`kubectl` 在 $HOME/.kube 目录中寻找一个名为 config 的文件。您可以通过设置环境变量 KUBECONFIG 或设置 [`--kubeconfig`](/zh/docs/concepts/configuration/organize-cluster-access-kubeconfig/) 参数指定其它 [kubeconfig](/zh/docs/concepts/configuration/organize-cluster-access-kubeconfig/) 文件。
+可以使用 Kubectl 命令行工具来管理 Kubernetes 集群。
```
[the remark this answers is not in the document]
Comment (yuandongx) [reply content/zh/docs/reference/kubectl/overview.md]: Thanks, done.

File: content/zh/docs/reference/kubectl/overview.md [hunk 17]
```diff
@@ -25,14 +25,27 @@ card:
...
+`kubectl` 在 $HOME/.kube 目录中查找一个名为 `config` 的配置文件。
```
[the remark this answers is not in the document]
Comment (yuandongx) [reply content/zh/docs/reference/kubectl/overview.md]: Thanks, done.

File: content/zh/docs/reference/kubectl/overview.md [hunk 18]
```diff
@@ -25,14 +25,27 @@ card:
...
-This overview covers `kubectl` syntax, describes the command operations, and provides common examples. For details about each command, including all the supported flags and subcommands, see the [kubectl](/docs/reference/generated/kubectl/kubectl-commands/) reference documentation. For installation instructions see [installing kubectl](/docs/tasks/kubectl/install/).
+This overview covers `kubectl` syntax, describes the command operations, and provides common examples.
+For details about each command, including all the supported flags and subcommands, see the
+[kubectl](/docs/reference/generated/kubectl/kubectl-commands/) reference documentation.
+For installation instructions see [installing kubectl](/docs/tasks/tools/install-kubectl/).
 -->
-本文概述了 `kubectl` 语法和命令操作描述，并提供了常见的示例。有关每个命令的详细信息，包括所有受支持的参数和子命令，请参阅 [kubectl](/docs/reference/generated/kubectl/kubectl-commands/) 参考文档。有关安装说明，请参见 [安装 kubectl](/zh/docs/tasks/tools/install-kubectl/) 。
+本文概述了 `kubectl` 语法和命令操作描述，并提供了常见的示例。
+有关每个命令的详细信息，包括所有受支持的参数和子命令，
+请参阅 [kubectl](/zh/docs/reference/generated/kubectl/kubectl-commands/) 参考文档。
```
[the remark this answers is not in the document]
Comment (yuandongx) [reply content/zh/docs/reference/kubectl/overview.md]: Thanks, done.

File: content/zh/docs/reference/kubectl/overview.md [hunk 19]
```diff
@@ -25,14 +25,27 @@ card:
...
+有关安装说明，请参见 [安装 kubectl](/zh/docs/tasks/tools/install-kubectl/) 。
```
[the remark this answers is not in the document]
Comment (yuandongx) [reply content/zh/docs/reference/kubectl/overview.md]: Thanks, done.

File: content/zh/docs/reference/kubectl/overview.md [hunk 20]
```diff
@@ -143,70 +156,102 @@ The following table includes short descriptions and the general syntax for all o
...
 `annotate`    | <code>kubectl annotate (-f FILENAME &#124; TYPE NAME &#124; TYPE/NAME) KEY_1=VAL_1 ... KEY_N=VAL_N [--overwrite] [--all] [--resource-version=version] [flags]</code> | 添加或更新一个或多个资源的注解。
+`api-resources`    | `kubectl api-resources [flags]` | 列出可用的 API 资源。
 `api-versions`    | `kubectl api-versions [flags]` | 列出可用的 API 版本。
 `apply`            | `kubectl apply -f FILENAME [flags]`| 从文件或 stdin 对资源应用配置更改。
 `attach`        | `kubectl attach POD -c CONTAINER [-i] [-t] [flags]` | 附加到正在运行的容器，查看输出流或与容器（stdin）交互。
+`auth`    | `kubectl auth [flags] [options]` | 检查授权。
 `autoscale`    | <code>kubectl autoscale (-f FILENAME &#124; TYPE NAME &#124; TYPE/NAME) [--min=MINPODS] --max=MAXPODS [--cpu-percent=CPU] [flags]</code> | 自动伸缩由副本控制器管理的一组 pod。
+`certificate`    | `kubectl certificate SUBCOMMAND [options]` | 修改证书资源。
 `cluster-info`    | `kubectl cluster-info [flags]` | 显示有关集群中主服务器和服务的端口信息。
+`completion`    | `kubectl completion SHELL [options]` | 为指定的 shell （bash 或 zsh）输出 shell 完成代码。
```
[the remark this answers is not in the document]
Comment (yuandongx) [reply content/zh/docs/reference/kubectl/overview.md:216]: Thanks, done.

File: content/zh/docs/reference/kubectl/overview.md [hunk 21]
```diff
@@ -143,70 +156,102 @@ The following table includes short descriptions and the general syntax for all o
...
 `config`        | `kubectl config SUBCOMMAND [flags]` | 修改 kubeconfig 文件。有关详细信息，请参阅各个子命令。
+`convert`    | `kubectl convert -f FILENAME [options]` | 在不同的API版本之间转换配置文件。配置文件可以是 YAML 或 JSON 格式。
```
[the remark this answers is not in the document]
Comment (yuandongx) [reply content/zh/docs/reference/kubectl/overview.md:218]: Thanks, done.

File: content/zh/docs/reference/kubectl/overview.md [hunk 22]
```diff
@@ -143,70 +156,102 @@ The following table includes short descriptions and the general syntax for all o
...
 `label`        | <code>kubectl label (-f FILENAME &#124; TYPE NAME &#124; TYPE/NAME) KEY_1=VAL_1 ... KEY_N=VAL_N [--overwrite] [--all] [--resource-version=version] [flags]</code> | 添加或更新一个或多个资源的标签。
 `logs`        | `kubectl logs POD [-c CONTAINER] [--follow] [flags]` | 在 pod 中打印容器的日志。
+`options`    | `kubectl options` | 全局命令行选项列表，适用于所有命令。
 `patch`        | <code>kubectl patch (-f FILENAME &#124; TYPE NAME &#124; TYPE/NAME) --patch PATCH [flags]</code> | 使用策略合并 patch 程序更新资源的一个或多个字段。
+`plugin`    | `kubectl plugin [flags] [options]` | 提供用于与插件交互的实用程序。
 `port-forward`    | `kubectl port-forward POD [LOCAL_PORT:]REMOTE_PORT [...[LOCAL_PORT_N:]REMOTE_PORT_N] [flags]` | 将一个或多个本地端口转发到一个 pod。
 `proxy`        | `kubectl proxy [--port=PORT] [--www=static-dir] [--www-prefix=prefix] [--api-prefix=prefix] [flags]` | 运行 Kubernetes API 服务器的代理。
 `replace`        | `kubectl replace -f FILENAME` | 从文件或标准输入中替换资源。
-`rolling-update`    | <code>kubectl rolling-update OLD_CONTROLLER_NAME ([NEW_CONTROLLER_NAME] --image=NEW_CONTAINER_IMAGE &#124; -f NEW_CONTROLLER_SPEC) [flags]</code> | 通过逐步替换指定的副本控制器及其 pod 来执行滚动更新。
+`rollout`    | `kubectl rollout SUBCOMMAND [options]` | 管理资源的部署。有效的资源类型包括：部署，守护程序集和状态集。
```
[the remark this answers is not in the document]
Comment (yuandongx) [reply content/zh/docs/reference/kubectl/overview.md:240]: Thanks, done.

File: content/zh/docs/reference/kubectl/overview.md (hunk 9 above)
Comment (yuandongx) [reply content/zh/docs/reference/kubectl/overview.md:245]: Thanks, done.

File: content/zh/docs/reference/kubectl/overview.md [hunk 23]
```diff
@@ -95,8 +108,8 @@ where `command`, `TYPE`, `NAME`, and `flags` are:
 
    * To specify resources with one or more files:  `-f file1 -f file2 -f file<#>`
 
-      * [Use YAML rather than JSON](/docs/concepts/configuration/overview/#general-config-tips) since YAML tends to be more user-friendly, especially for configuration files.<br/>
-     Example: `kubectl get pod -f ./pod.yaml`
+      * [Use YAML rather than JSON](/docs/concepts/configuration/overview/#general-configuration-tips) since YAML tends to be more user-friendly, especially for configuration files.<br/>
```
[the remark this answers is not in the document]
Comment (yuandongx) [reply content/zh/docs/reference/kubectl/overview.md:111]: Thanks, done.

**Comment by yuandongx** [comment]
> Why is this closed? I have spent about one hour reviewing it ...

Sorry for that .... It seem that there was something error whe I tried to squash commit.

File: content/zh/docs/reference/kubectl/overview.md (hunk 10 above)
Comment (tengqm) [inline content/zh/docs/reference/kubectl/overview.md:691]: 706 行的命令参数已经不存在。

File: content/zh/docs/reference/kubectl/overview.md [hunk 24]
```diff
@@ -749,71 +813,80 @@ kubectl logs -f <pod-name>
...
 kubectl hello
 ```
 -->
 ```shell
 # 用任何语言创建一个简单的插件，并为生成的可执行文件命名
 # 以前缀 "kubectl-" 开始
 cat ./kubectl-hello
+```
+
+```shell
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/overview.md:847]: line 848 bash -> sh

File: content/zh/docs/reference/kubectl/overview.md [hunk 25]
```diff
@@ -822,27 +895,27 @@ The following kubectl-compatible plugins are available:
 /usr/local/bin/kubectl-bar
 ```
 -->
+输出类似于：
 ```
-以下 kubectl-适配 的插件是可用的：
+以下 kubectl-适配的插件是可用的：
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/overview.md:900]: 900 行保留英文原文，是 kubectl 命令的输出。

File: content/zh/docs/reference/kubectl/overview.md [hunk 26]
```diff
@@ -822,27 +895,27 @@ The following kubectl-compatible plugins are available:
...
 -->
-```
-# 这个指令也可以警告我们哪些插件
+```shell
+# `kubectl plugin list`指令也可以向你告警哪些插件
 # 被运行，或是被其它插件覆盖了
-# 例如
-sudo chmod -x /usr/local/bin/kubectl-foo
+# 例如:
+sudo chmod -x /usr/local/bin/kubectl-foo # 删除执行权限
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/overview.md:918]: line 914-917 不是 shell 的一部分，也不用加 "#"

File: content/zh/docs/reference/kubectl/overview.md [hunk 27]
```diff
@@ -858,7 +931,7 @@ error: one plugin warning was found
 ```
 -->
 ```
-以下 kubectl-适配 的插件是可用的：
+以下 kubectl- 适配 的插件是可用的：
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/overview.md:934]:
```suggestion
以下 kubectl 兼容的插件是可用的：
```

File: content/zh/docs/reference/kubectl/overview.md [hunk 28]
```diff
@@ -869,55 +942,66 @@ error: one plugin warning was found
 ```
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/overview.md:942]: 941 行保留原文，934 行保留原文。
这些是 kubectl 的输出，不要翻译。

File: content/zh/docs/reference/kubectl/overview.md [hunk 29]
```diff
@@ -869,55 +942,66 @@ error: one plugin warning was found
...
+-->
+```shell
 cat ./kubectl-whoami
+```
+
+<!--
+The next few examples assume that you already made `kubectl-whoami` have
+the following contents:
+-->
+接下来的几个示例假设你已经将“kubectl whoami”设置为以下内容:
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/overview.md:962]:
```suggestion
接下来的几个示例假设你已经将 `kubectl-whoami` 设置为以下内容:
```

File: content/zh/docs/reference/kubectl/overview.md [hunk 30]
```diff
@@ -869,55 +942,66 @@ error: one plugin warning was found
...
 -->
-
 ```shell
-cat ./kubectl-whoami
 #!/bin/bash
 
-# 这个插件借用 `kubectl config` 指令来输出
-# 当前用户的信息，基于当前指定的 context
-kubectl config view --template='{{ range .contexts }}{{ if eq .name "'$(kubectl config current-context)'" }}Current user: {{ .context.user }}{{ end }}{{ end }}'
+# 这个插件使用`kubectl config`命令的输出
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/overview.md:976]:
```suggestion
# 这个插件利用 `kubectl config` 命令基于当前所选上下文输出当前用户的信息
```

File: content/zh/docs/reference/kubectl/overview.md [hunk 31]
```diff
@@ -869,55 +942,66 @@ error: one plugin warning was found
...
+# 基于当前所选的上下方有关当前用户信息
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/overview.md:977]: 删除 977

File: content/zh/docs/reference/kubectl/overview.md [hunk 32]
```diff
@@ -869,55 +942,66 @@ error: one plugin warning was found
...
 ```
 
 <!--
-Running the above plugin gives us an output containing the user for the currently selected
-context in our KUBECONFIG file:
+Running the above command gives you an output containing the user for the
+current context in your KUBECONFIG file:
 -->
-运行上面的插件为我们提供了一个输出，其中包含我们 KUBECONFIG 文件中当前所选定上下文对应的用户：
+运行以上命令将为你提供一个输出，其中包含KUBECONFIG文件中当前上下文的用户:
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/overview.md:985]:
```suggestion
运行以上命令将为你提供一个输出，其中包含 KUBECONFIG 文件中当前上下文的用户:
```

File: content/zh/docs/reference/kubectl/overview.md [hunk 33]
```diff
@@ -935,6 +1019,10 @@ To find out more about plugins, take a look at the [example cli plugin](https://
 
 
 <!--
-Start using the [kubectl](/docs/reference/generated/kubectl/kubectl-commands/) commands.
+* Start using the [kubectl](/docs/reference/generated/kubectl/kubectl-commands/) commands.
+
+* To find out more about plugins, take a look at the [example cli plugin](https://github.com/kubernetes/sample-cli-plugin).
 -->
-开始使用 [kubectl](/docs/reference/generated/kubectl/kubectl-commands/) 命令。
+* 开始使用 [kubectl](/zh/docs/reference/generated/kubectl/kubectl-commands/) 命令。
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/overview.md:1026]:
```suggestion
* 开始使用 [kubectl](/docs/reference/generated/kubectl/kubectl-commands/) 命令。
```

**Comment by tengqm** [comment]
Thanks for the hard work!
/lgtm
/approve

**Pull request merged**